pkg/iac/scanners/terraformplan/json/scanner.go

```diff
@@ -1,4 +1,4 @@
-package terraformplan
+package json
```

Comment:
can we rename this to tfjson to avoid confusion with the stdlib package?
```suggestion
package tfjson
```

Reply:
Wouldn't this cause confusion with the JSON configuration syntax? https://developer.hashicorp.com/terraform/language/syntax/json